bookwyrm/activitypub/base_activity.py

```diff
@@ -194,6 +194,11 @@ def serialize(self, **kwargs):
             try:
                 if issubclass(type(v), ActivityObject):
                     data[k] = v.serialize()
+                elif isinstance(v, list):
+                    data[k] = [
```

Comment:
Directly nested lists are not allowed in JSON-LD, so I think we should be alright!